schematic/models/validate_attribute.py

```diff
@@ -594,9 +758,19 @@ def list_validation(
             list_robustness = "strict"
 
         if list_robustness == "strict":
+            manifest_col = manifest_col.astype(str)
+
             # This will capture any if an entry is not formatted properly. Only for strict lists
             for i, list_string in enumerate(manifest_col):
-                if not re.fullmatch(csv_re, list_string):
+                list_error = None
+                entry_has_value = self.get_entry_has_value(
+                    entry=list_string,
+                    node_display_name=manifest_col.name,
+                )
+
+                if not isinstance(list_string, str) and entry_has_value:
+                    list_error = "not_a_string"
```

Comment:
Whats is the purpose of this if block? `list_error` only gets used in the next elif block